docs/pages/reference/mesheryctl-commands.md

```diff
@@ -117,13 +117,26 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
       </tr>
     {% endfor %}
   {% endfor %}
-    {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system.subcommands %}{% assign subcommand = subcommand_hash[1] %}
+  {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system.config.command %}{% assign subcommand = subcommand_hash[1] %}
```

Comment:
Also, the preview for your PR doesn't show any link to the config page

![image](https://user-images.githubusercontent.com/23498248/119465060-f59ab900-bd60-11eb-8036-68cbd12717a0.png)

Reply:
I just saw that, but when I am running it locally it runs just fine.

Reply:
If you don't mind could you share a SS?

Reply:
![Screenshot from 2021-05-25 14-22-10](https://user-images.githubusercontent.com/54525960/119469473-02b9a700-bd65-11eb-8a0f-718259dbaaf4.png)
![Screenshot from 2021-05-25 14-23-00](https://user-images.githubusercontent.com/54525960/119469477-03ead400-bd65-11eb-99e4-1ddfa8844b1f.png)

Reply:
I guess there is one file missing here the one containing meshery-sys-config.yml but I am not able to understand how can that happen as in my local machine it shows tree is clean. Shall I close this PR and create a new one?

Reply:
![Screenshot from 2021-05-25 15-13-26](https://user-images.githubusercontent.com/54525960/119476626-d8b7b300-bd6b-11eb-96f5-6b80f7245d11.png)

Reply:
I guess the file has been committed, could you run `git push origin <your_branch_name>`

Reply:
Also the commit doesn't exist in your repo also, https://github.com/Blakelist7/meshery/tree/config-page/docs/_data/mesheryctlcommands/lifecycle/system

Reply:
Try editing something in the file and then commit, push it

Reply:
Done

Reply:
I don't know why git was showing directory named config, so had to change that name.
Do you have any idea?